hipcheck/src/session/pm.rs

```diff
@@ -317,13 +316,31 @@ fn extract_package_version_from_url(url: Url) -> Result<(String, String)> {
 	// Get the package and version from the URL
 	if package_type.contains(NPM) {
 		//npm gets the first two segments
-		let (package, version) = url
+		// let (package, version) = url
+		// 	.path_segments()
+		// 	.map(|mut i| (i.next(), i.next()))
+		// 	.ok_or_else(|| hc_error!("can't detect package name"))?;
+		// Ok((
+		// 	package.unwrap().to_string(), //this will graceful error if empty because of the mapping above I believe
+		// 	version.unwrap_or("no version").to_string(),
+		// ))
```

Comment:
Is there a reason to want to keep this old code around? If not we ought to remove it.

Reply:
No, I think I just forgot to remove it. :) I can take it out and make another squashed commit tonight.